sources/packages/backend/libs/test-utils/src/factories/institution-location.ts

```diff
@@ -2,8 +2,16 @@ import { Institution, InstitutionLocation } from "@sims/sims-db";
 import * as faker from "faker";
 import { createFakeInstitution } from "./institution";
 
+/**
+ * Create fake institution location.
+ * @param institution institution.
+ * @param options institution location options.
+ * - `institutionCode` institution code.
+ * @returns institution location
+ */
 export function createFakeInstitutionLocation(
   institution?: Institution,
+  options?: { institutionCode?: string },
```

Comment:
It can also be done on the consumer's part. Just let the random number be created and then override it outside.
Just trying to keep the idea that we have been following, I guess.